testConnectivity.py

```diff
@@ -236,7 +239,17 @@ def testConnectivity(args):
             pass
 
         vfatBoard.parentOH.parentAMC.writeRegister("GEM_AMC.TTC.GENERATOR.ENABLE",0x0)
-        
+
+        print("Checking trigger link status:")
+        for trial in range(0,args.maxIter ):
+            testLinks = vfatBoard.parentOH.parentAMC.getTriggerLinkStatus(printSummary=True, checkCSCTrigLink=False, ohMask=args.ohMask)
```

Comment:
It seems this only partially addresses #224 since the `checkCSCTrigLink` argument to `HwAMC::getTriggerLinkStatus` is hardcoded to false.

I think we need to a command line option (see under `Concerns for Only QC7` under #224).